### PR TITLE
Some fixes, hopefully, for DCM feedback errors found on test data

### PR DESCRIPTION
**Pull request by eugenpt**
When testing on DCM dataset the following issue happens on ~scan 161, after the first FB session is evaluated

(here I emulated online session by copying files one by one every TR s, to exclude possible offline mode issues; so the output somewhat differs from offline mode)

```
2022-03-15 07:35:05 | INFO      | opennft:callMainLoopIteration - Call iteration for file "001_000008_000161.dcm"
TIMING: 5 iter - PREPROC MC: 2.534752e-01 s - SMOOTH: 2.042950e-02 s - IGLM: 5.246740e-02 s
2022-03-15 07:35:06 | INFO      | opennft:callMainLoopIteration -   preprocess fMRI Volume: 0.36 s
2022-03-15 07:35:06 | INFO      | opennft:callMainLoopIteration -   preprocess signal: 17.00 ms
2022-03-15 07:35:06 | INFO      | opennft:callMainLoopIteration - Sending by UDP - dispValue = 0.0
2022-03-15 07:35:06 | INFO      | opennft:displayMosaicImage - Getting background volume: 7.00 ms
2022-03-15 07:35:06 | INFO      | opennft:callMainLoopIteration - Display mosaic image: 22.00 ms
Traceback (most recent call last):
  File "opennft-venv\lib\site-packages\opennft\opennft.py", line 978, in callMainLoopIteration
    self.drawRoiPlots(init)
  File "opennft-venv\lib\site-packages\opennft\opennft.py", line 2685, in drawRoiPlots
    (dataNorm, np.array([self.displaySamples]) / self.P['MaxFeedbackVal'])
  File "opennft-venv\lib\site-packages\pyqtgraph\numpy_fix.py", line 13, in concatenate
    return np.concatenate_orig(vals, *args, **kwds)
  File "<__array_function__ internals>", line 6, in concatenate
ValueError: all the input array dimensions for the concatenation axis must match exactly, but along dimension 1, the array at index 0 has size 109 and the array at index 1 has size 151
```

So at the time the shapes of arrays are:
```
dataNorm.shape:
(3, 109)
np.array([self.displaySamples]).shape
(1, 151)
```

The first commit of this PR seems to solve the issue, however I am not fully sure this is the proper way of doing that.

Also the second commit is about feedback always being 0, but that is a much clearer issue or at least I think it is.

If the issue should not be hapening anyway, and is the result of local setup and not OpenNFT, please let me know, in any case feedback is apreciated

### CODE REVIEW

**Comment by eugenpt** [comment]
Ok, so the issue is about `plot feedback` option neither working nor actually needed for DCM feedback. If it is off - no error, naturally

**Comment by TheAmid** [comment]
In case when `PlotFeedback` is on for DCM, I suggest not drawing this zero-line on main plot. 
May be it is possible to add additional flag on line 2678 in opennft.py 
`if self.P['PlotFeedback']:
            dataNorm = np.concatenate(
                (dataNorm, np.array([self.displaySamples]) / self.P['MaxFeedbackVal'])
            )`

**Comment by eugenpt** [comment]
I suggest also disabling the option checkbox on DCM DB Type.

Also - maybe for trial-based feedback the feedback plot should just be numbers displayed on corresponding time positions as text ?

File: opennft/opennft.py [hunk 1]
```diff
@@ -2676,7 +2685,7 @@ def drawRoiPlots(self, init):
         dataRaw = np.array(self.outputSamples[key], ndmin=2)[self.selectedRoi,:]
         dataProc = np.array(self.outputSamples['kalmanProcTimeSeries'], ndmin=2)[self.selectedRoi,:]
         dataNorm = np.array(self.outputSamples['scalProcTimeSeries'], ndmin=2)[self.selectedRoi,:]
-        if self.P['PlotFeedback']:
+        if self.P['PlotFeedback'] and self.P['Type'] != 'DCM':
```
Comment (TheAmid) [inline opennft/opennft.py:2689]: I guess, it is unnecessary to add this flag if you disable PlotFeedback in `onChangeFBType` method.

I checked the rest and it's fine.

**Comment by eugenpt** [comment]
> As for displaying numbers on the plot, I think it will be hard to see. We can, e.g.` fprintf(['dispValue ' mat2str(dispValue) '\n']);` from displayFeedback.m, which will put them to log (easy searchable in Pycharm during real-time).

But why would they be hard to see? I expect ~10 trials per run, why 10 numbers would be hard to see? not that much harder than any plot ticks or scan markers

**Pull request merged**